packages/typescript/src/internal/devserver/launcher_template.sh

```diff
@@ -42,8 +42,45 @@ else
 fi
 # --- end runfiles.bash initialization ---
 
+# Check environment for which node path to use
```

Comment:
maybe we could key this off the path to node instead?

Reply:
Hmm, maybe you can consider this:

Adding the following to .bazelrc

```
--enable_platform_specific_config
build:windows --action_env=NODE_PLATFORM=windows
build:macos --action_env=NODE_PLATFORM=macos
build:linux --action_env=NODE_PLATFORM=linux
```

Then you can check the value of `ctx.configuration.default_shell_env['NODE_PLATFORM']`

Doc for [--enable_platform_specific_config](--enable_platform_specific_config)

Reply:
Oh, sorry. I commented at the wrong place.